Extensions/blacklist.js

```diff
@@ -561,8 +567,8 @@ XKit.extensions.blacklist = new Object({
 					m_content = $(this).find(".caption").html();
 				}
 
-				if ($(this).find(".reblog-list-item").length > 0) {
-					m_content = $(this).find(".reblog-list-item").map(function() {
+				if ($(this).find(".reblog-list-item, .reblog-content").length > 0) {
```

Comment:
i checked again and mobile also uses reblog-list-item class, so this change will be reverted